config/build/rollup.config.js

```diff
@@ -222,13 +292,10 @@ const getPackageInfo = (commandLineArgs, pkgName) => {
 /**
  * Creates the full rollup configuration for the given args.
  */
-const createRollupConfig = (commandLineArgs) => {
-    // Determine what packages we are building.
-    const pkgNames = fs.readdirSync("packages");
-
+const createRollupConfig = async (commandLineArgs) => {
     // For the packages we have determined we want, let's get more information
-    // about them and  generate configurations.
-    return pkgNames
+    // about them and generate configurations.
+    return getPackageDirNamesInBuildOrder()
```

Comment:
Now that we yarn workspaces, I don't think we need to 'dir' the packages directory to figure out what to build... yarn handles that.